cypress/support/commands.ts

```diff
@@ -14,23 +14,44 @@ import { Result } from "axe-core";
 import "@cypress/code-coverage/support";
 import "cypress-axe";
 
+const loginWithRetry = (iteration: number = 0): void => {
+    if (iteration >= 4) {
+        // This only ever seems to happen on GH Actions on the first test, so we can just log and move on
+        // Redirects are tested by the auth spec
+        cy.log("Login redirect failed");
+        return;
+    }
+
+    cy.get("h1").then((heading) => {
+        if (heading.text().includes("Login")) {
+            cy.get("button[type='submit']").then((submitButton) => {
+                if (submitButton.not(":disabled")) {
+                    submitButton.trigger("click");
```

Comment:
is this the same as `submitButton.click()`?

Reply:
No, if the button is disabled and we try `submitButton.click()`, it would result in a test failure.

Reply:
submitButton is a jquery element, and `click` is deprecated in jquery in favour of `.trigger("click")`

I could also do `cy.wrap(submitButton).click()` if you prefer that?